utils/documentOps.py

```diff
@@ -142,6 +174,26 @@ def print_row(array):
         str += a + " |"
     print(str)
 
+# Min opset: if explicit ==MIN== value, use as is. Otherwise min of default and
+# custom versions delivered in opset_dict.
+def compute_min_opset(op):
+    if op in min_dict:
+        return min_dict[op]
+    min_list = [min_opset_default]
```

Comment:
My thoughts are the same for min_opset... I do not think we should hardcode it, it would be safer to use the smallest or last number in the list for `version_dict`. From my exercise, not all operations even support a version 9, so this would be misleading.